doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst

```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
-This way, you can provide Wi-Fi capabilities to develop Wi-Fi applications with another System on Chip (SoC), MPU, or MCU host.
-For the pinout of the castellated holes, see `nRF7002 EB castellated edge holes`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
+
+For nRF54H20 DK and for nRF54L15 PDK following are the pin mapping:
+
+.. tabs::
+
+   .. group-tab:: nRF54H20 DK
+
+       The pin mapping for the nRF54H20 DK is as follows:
+
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| nRF70 Series pin name (EB name)   | nRF54H20 DK pins  | Function                                      |
+		+===================================+===================+===============================================+
+		| CLK (CLK)                         | P1.01             | SPI Clock                                     |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| CS (CS)                           | P1.04             | SPI Chip Select                               |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| MOSI (D0)                         | P1.05             | SPI MOSI                                      |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| MISO (D1)                         | P1.06             | SPI MISO                                      |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| BUCKEN (EN)                       | P1.00             | Enable Buck regulator                         |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| IOVDDEN (Not used)                | P1.00             | Enable IOVDD regulator                        |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| IRQ (IRQ)                         | P1.02             | Interrupt from nRF7002                        |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| GRANT (GRT)                       | P1.03             | Coexistence grant from nRF7002                |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| REQ (REQ)                         | P1.08             | Coexistence request to nRF7002                |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| STATUS (ST0)                      | P1.07             | Coexistence status from nRF7002               |
+		+-----------------------------------+-------------------+-----------------------------------------------+
```

Comment:
I think we don't need the indent here and we can align the table edge to `group-tab`?

Reply:
Table indent seems to be tabs instead of spaces as well, which isn't what we generally use.